### PR TITLE
Update Component client command to use new contact API.

### PR DESCRIPTION
**Pull request by xychu**
Will ignore old contacts if any.
JIRA: PDC-1166

### CODE REVIEW

File: pdc_client/plugins/component.py [hunk 1]
```diff
@@ -70,6 +70,14 @@ def global_component_info(self, args, global_component_id=None):
         global_component_id = global_component_id or args.global_component_id
         global_component = self.client['global-components'][global_component_id]._()
 
+        component_contacts = get_paged(self.client['global-component-contacts']._, component=global_component['name'])
+        contacts = []
+        if component_contacts:
```
Comment (erichuanggit) [inline pdc_client/plugins/component.py]: This code looks like almost same as what you did for `release-component-contacts`. How about abstract it out to a single function avoiding code duplicated?

Comment (xychu) [inline pdc_client/plugins/component.py]: Sure, will update later.

**Comment by lubomir** [comment]
Given that the logic in formatting contacts required a separate function, It seems like a good idea to have some tests for showing components with contacts.

**Comment by xychu** [comment]
@lubomir good point and done.

**Comment by erichuanggit** [comment]
Looks good to me.

**Comment by lubomir** [comment]
:+1:

**Pull request merged**